scripts/package.js

```diff
@@ -104,7 +104,7 @@ function main() {
     'ducks/nodeReducer.js',
     'ducks/hip2Reducer.js',
     'ducks/walletReducer.js',
-    'ducks/claims.js'
+    'ducks/notifications.js'
```

Comment:
Let's add a new line here instead of replacing claims. And can you add a new line in package.sh similarly (duplicate and edit the last line)? That's used for Linux and Macos.